src/reason-parser/reason_lexer.mll

```diff
@@ -595,6 +587,25 @@ rule token = parse
   *)
   | "}"  { RBRACE }
   | ">}" { GREATERRBRACE }
+  | "=<" {
```

Comment:
What would happen if someone defines an infix operator `=<`?

```reason
let (=<) = (a, b) => a + b;
let x = 10;
let y = 20;
let result = x =< y;
```
Today, the result is `30`. After this change, what happens?

I think it would parse it as `LET ident EQUAL ident EQUAL LESSTHAN ident`, which is not what we want.

I think what you can do is be more specific here, and only do this lexing backtracking in the case that you have `=<ident` without any whitespace. If you do that, then it would mean that after the change, we could still parse:

```reason
let result = x =< y;
```

But we couldn't parse:

```reason
let result = x=<y;
```

Because there's no white space between the `<` and the identifier `y`.

I think it's fine to say "=<` as an infix operator must have white space after the `<`, but not fine to say "`=<` is not longer a valid infix operator".


If I'm wrong about this diff, perhaps we could have some test cases demonstrating that it's not an issue right now.